s3/pkg/service/bucket.go

```diff
@@ -95,7 +95,6 @@ func (s *s3Service) CreateBucket(ctx context.Context, in *pb.Bucket, out *pb.Bas
 			return err
 		}
 	}
-	return nil
```

Comment:
Why delete 'return nil' here?

Reply:
because the code after this became unreachable...

Reply:
Ok, I missed the last return.